src/lightning_app/utilities/layout.py

```diff
@@ -79,11 +81,17 @@ def _collect_content_layout(layout: List[Dict], flow: "lightning_app.LightningFl
                 f" For the value, choose either a reference to a child flow or a URla."
             )
         if isinstance(entry["content"], str):  # assume this is a URL
-            # The URL isn't fully defined yet. Looks something like ``self.work.url + /something``.
-            if entry["content"].startswith("/"):
+            url = entry["content"]
+            if url.startswith("/"):
+                # The URL isn't fully defined yet. Looks something like ``self.work.url + /something``.
                 entry["target"] = ""
             else:
-                entry["target"] = entry["content"]
+                entry["target"] = url
+            if url.startswith("http") and not url.startswith("https") and is_running_in_cloud():
+                warnings.warn(
+                    f"You configured an http link {url[:32]}... but it may not be accessible in the cloud."
```

Comment:
It will _never_ be accessible in the cloud due to `https`/`http` restrictions by modern browsers.

Reply:
Got it, updated the text.